lib/menu.rb

```diff
@@ -0,0 +1,40 @@
+require_relative './viewer'
+require_relative './dish'
+require_relative './cart'
+require_relative './sms_sender'
+
```

Comment:
This class has a few dependencies – could you inject them to make the solution more flexible and easier to test?